apm-collector/apm-collector-boot/src/main/resources/component-libraries.yml

```diff
@@ -122,9 +122,30 @@ Jedis:
   languages: Java
 
 
+# .NET/.NET Core components
+AspNetCore:
+  id: 1001
```

Comment:
I prefer to keep  [0, 3000) for Java and across language frameworks. So please use [3000, 4000) for C#/.NET only.